src/app/product-details-page/components/product-description/product-image-preview/product-image-preview.component.spec.ts

```diff
@@ -4,14 +4,15 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { ProductDescriptionComponent } from '../product-description.component';
 import { ProductOrderComponent } from '../product-order/product-order.component';
 import { ZoomComponent } from './zoom/zoom.component';
+import { LazyLoadComponent } from 'src/app/shared/components/lazy-load/lazy-load.component';
 
 describe('ProductImagePreviewComponent', () => {
   let component: ProductImagePreviewComponent;
   let fixture: ComponentFixture<ProductImagePreviewComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ProductImagePreviewComponent, ProductDescriptionComponent, ProductOrderComponent, ZoomComponent],
+      declarations: [ProductImagePreviewComponent, ProductDescriptionComponent, ProductOrderComponent, ZoomComponent ],
```

Comment:
When you have an array spreading over several lines, I think it's better to have each item on a separate line, like this:
```
declarations: [
  ProductImagePreviewComponent,
  ProductDescriptionComponent,
  ProductOrderComponent,
  ZoomComponent
],
```